packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/pathes.dart

```diff
@@ -155,18 +135,11 @@ class _ClassStatsTableState extends State<ClassStatsTable>
       columns: _columns,
       data: widget.data.entries,
       keyFactory: (e) => Key(e.key.asLongString()),
+      // We want sorting to be the same for all snapshots and classes.
+      dataKey: '',
```

Comment:
give it a key like 'ClassStatsTable' then. This will uniquely identify this table so that the sorting is stored and looked up by 'ClassStatsTable'